### ISSUE TITLE
Devices

### ISSUE THREAD

**Issue by BaxterGarr** [Acidham/alfred-airpod-connector#24]
I am only showing one set of Airpods when I have multiple connected to the Bluetooth. I cannot remember where I was able to set the devices the workflow picks up or how to have it see the other ones. Any suggestions? Thanks

**Comment by Acidham** [comment]
The workflow basically picks all connected AirPods to your account/computer.  When you don't see them, are those visible under Bluetooth in system settings?

**Comment by BaxterGarr** [comment]
Yes, they are visible in the bluetooth on my machine. Should I try to update that step or reinstall?

**Comment by Acidham** [comment]
can you please execute `system_profiler SPBluetoothDataType -json` in terminal and provide me with the response? Ensure to mask whatever you want.
